### ISSUE TITLE
Impelement an HTTP Server

### ISSUE THREAD

**Issue by nsoblath** [driplineorg/dripline-python#171]
Nothing here yet; just adding the issue to track the work

**Comment by allgoodpnw** [comment]
so far, I've gone through the "First Mesh Walkthrough" tutorial. Was able to successfully execute the key-value-store container. So far, the dl-agent cli has the following subcommands:
- get
  - works. tested using the command provided in the tutorial to get `peaches`
- set
  - works as intended

@nsoblath Im curious to hear your thoughts on why the dl-agent needs to support HTTP requests `PUT` and `POST`. Do we actually care about that distinction in our API?

**Comment by nsoblath** [comment]
> @nsoblath Im curious to hear your thoughts on why the dl-agent needs to support HTTP requests `PUT` and `POST`. Do we actually care about that distinction in our API?

I guess what I really care about is supporting the types of message some front end that knows how to send HTTP requests would send.  For instance, if someone built a web GUI with some relatively modern tools, what would make the most sense?  My initial plan where I matched `PUT` HTTP requests to dripline set requests and `POST` HTTP requests to dripline cmd requests was just based on my understanding of the purposes of those HTTP request types.

**Comment by allgoodpnw** [comment]
This ticket is pretty much completed but has been blocked by unit tests breaking that seem to be unrelated to the changes made to the repo. I'll add a link to another issue i create to fix unit tests
